uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/leave-request-ctrl_test.js

```diff
@@ -1254,23 +1254,6 @@
         });
```

Comment:
`spyEntitlementAPI`should be removed which was introduced in https://github.com/civicrm/civihr/pull/1857/files.